src/test/kotlin/com/igorwojda/list/capitalizeFirst/challenge.kt

```diff
@@ -3,9 +3,13 @@ package com.igorwojda.list.capitalizeFirst
 import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
-private fun capitalizeFirst(list: List<String>): List<String> {
-    TODO("not implemented")
-}
+private fun capitalizeFirst(list: List<String>): List<String> =
+    if(list.size==0){
```

Comment:
challenge: implement this using a collection mapping function instead of a recursive function

Reply:
list.map{it.capitalize()}